src/utils/apiUtils.js

```diff
@@ -162,11 +162,22 @@ const pipelineDetails = {
   },
 }
 
+function trimName(srcDsName, tgtDsName, suffix) {
+  const limit = 70 - suffix.length - 4
```

Comment:
We could put `70` and `4` in variables so it's clear what they are meant for and make them easier to maintain. Also add a comment about what this function is doing and why it is important. @plinnegan

Reply:
@jimiolaniyan sorted now.